src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <>
+    <App />
+  </>,
```

Comment:
App 상위에 불필요한 wrapping이 있는것 같아요. 제거해도 좋을 것 같습니다!
```suggestion
    <App />,
```

Reply:
네 수정했습니다